game/forms.py

```
@@ -29,27 +57,34 @@ def clean(self):
-                context, 
-                from_email, 
+                subject_template_name,
+                email_template_name,
+                context,
+                from_email,
                 to_email,
-                html_email_template_name)
+                html_email_template_name,
+            )
+
         except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)